projects/Mallard/src/constants.ts

```diff
@@ -25,4 +29,7 @@ export {
     FACEBOOK_CLIENT_ID,
     GOOGLE_CLIENT_ID,
     AUTH_TTL,
+    LEGACY_SUBSCRIBER_ID_USER_DEFAULT_KEY,
+    LEGACY_SUBSCRIBER_POSTCODE_USER_DEFAULT_KEY,
+    LEGACY_CAS_EXPIRY_USER_DEFAULTS_KEY,
```

Comment:
this is a note for myself in the future, not this PR, but there's some stuff on `helpers/words` that is more better suited for this file!

Reply:
I love the idea of leaving notes around PRs, I'm going to do more of this 😆